ctapipe/reco/ImPACT.py

```diff
@@ -19,20 +19,41 @@
 from ctapipe.reco.reco_algorithms import Reconstructor
 from ctapipe.utils import TableInterpolator
 from ctapipe.instrument import get_atmosphere_profile_functions
+from ctapipe.reco.shower_max import ShowerMaxEstimator
+
 from scipy.optimize import minimize, least_squares
 from scipy.stats import norm
 
 __all__ = ['ImPACTReconstructor', 'energy_prior', 'xmax_prior']
 
 
+def guess_shower_depth(energy):
+    """
+    Simple estimation of depth of shower max based on the expected gamma-ray elongation 
+    rate.
+    
+    Parameters
+    ----------
+    energy: float
+        Energy of the shower in TeV
+        
+    Returns
+    -------
+    float: Expected depth of shower maximum
+    """
+    x_max_exp = 300 * (u.g*u.cm**-2) + \
```

Comment:
Thats actually a good question :-), I pulled these numbers from my thesis, but they are actually wrong (I accidentally took the values for electrons). However, this does not actually change the result as it is just used to define the binning used in the template generation, I think for now I will comment that this is an incorrect value and needs to be updated. Then I can add a proper reference when I next update the image templates.